src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -377,28 +377,30 @@ def __init__(self, handler: Any) -> None:
         self.handler = handler
         self.level_specs: Dict[str, LevelSpec] = {}
         self.schedules: Dict[str, Schedule] = {}
+        # Previous versions incorrectly stored the global config in
+        # the localized module option. Check the config is here and fix it.
+        self.replace_local_with_global_module_option()
```

Comment:
nit: I don't see much point in a function that is going to be used only once, the inline code would look good to me. But I am fine with the current code too.